docs/source/apps/projinfo.rst

```diff
@@ -244,7 +246,7 @@ Output:
     PROJ.4 string:
     +proj=longlat +datum=WGS84 +no_defs +type=crs
 
-    WKT2_2018 string:
+    WKT2_2019 string:
```

Comment:
> Does the standard say anything about the correct form of the name?

Good question. I'm not sure. The doc is there http://docs.opengeospatial.org/is/18-010r7/18-010r7.html and is referenced from https://www.opengeospatial.org/standards/wkt-crs . So it could be "WKT CRS 2.0" (because WKT2:2015 was "WKT CRS 1.0" ...), which would be super confusing. A better name could be "ISO 19162:2019" but that's probably too many figures for people to remember...  I wouldn't mind changing to "WKT2:2019"

Reply:
Okay, it's not a big deal, i just wondered. We use ISO19111:2019 to mark that we are talking about the 2019 version of the standard, so WKT2:2019 would follow the same pattern. Incidentally I also think it looks better than WKT2_2019 but that is just my personal preference.

Reply:
ok, just changed to WKT2:2019 (and WKT2:2015, WKT1:GDAL, WKT1:ESRI for consistency)

Reply:
Did you forget to push the change? The line above is still "WKT2_2019" as well as most occurrences below.

Reply:
ah, I didn't indeed change the .rst files. Now fixed. I've also changed the usage message of projinfo to use WKT2:2019. The utilty still accepts colon, underscore or dash as separator as it did before